mysql-test/r/sp-bugs.result

```diff
@@ -464,3 +464,30 @@ Function	sql_mode	Create Function	character_set_client	collation_connection	Data
 очень_очень_очень_очень_очень_очень_очень_очень_длинная_строка_e	#	#	#	#	#
 DROP FUNCTION очень_очень_очень_очень_очень_очень_очень_очень_длинная_строка_é;
 SET NAMES default;
+#
+# PS-6094 - Handler fails to trigger on Error 1049 or SQLSTATE 42000 or plain sqlexception
+#
+CREATE FUNCTION handler_func() RETURNS int(11)
+BEGIN
+DECLARE EXIT HANDLER FOR SQLEXCEPTION RETURN NULL;
+SELECT fake_col into @a FROM fake_table;
+RETURN 1;
+END|
+Warnings:
+Warning	1681	Integer display width is deprecated and will be removed in a future release.
```

Comment:
`RETURNS INTEGER` to silence this warning

Reply:
done